build.gradle.kts

```diff
@@ -42,6 +43,11 @@ allprojects {
 }
 
 subprojects {
+  apply(plugin = "org.jetbrains.kotlin.multiplatform")
+  configure<KotlinMultiplatformExtension> {
+    jvm {}
+  }
```

Comment:
Please apply and configure each project individually. I believe only the core artifact should be multiplatform. The others should all remain on `.jvm` plugin as they are fundamentally tied to the JVM in some way (KSP, JavaPoet, etc.).

Reply:
I've made the changes, but it seems like we're encountering issues with sections like `configure<KotlinProjectExtension>` and `tasks.withType<KotlinCompile>`, which fail due to the absence of the plugin. We might need to resort to code duplication, or we could establish a common configuration using `build-logic`. Do you have a preference for how we should proceed?

Reply:
Sorry, I might be able to use `plugins.withId`.